GA-worksheet.md

```diff
@@ -25,15 +20,15 @@
 
 ##Project #2 Part #2: Style the Header with CSS
 
-1. What advantages does the `<link>` tag have over the `<style>` tag?
+1. What advantages does the `<link>` tag have over the `<style>` tag?A link between a document and an external resource but the style just changes the color
 
-2. What is normalize.css?
+2. What is normalize.css?Small CSS file that provides better cross-browser consistency in the default styling of HTML elements
 
-3. Why do we need to add a padding property to the `<ul>` tag?
+3. Why do we need to add a padding property to the `<ul>` tag? So then the size wouldn't be too big or too small. 
```

Comment:
Take a Look at Project 2, Section 2, slide 14 in the dash tutorial. It's not about size in particular, but we want to get the nav bar centered.